content/ending-frozen-applications.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Closing Frozen Applications and Desktops
+description: >
+  Linux provides several methods to kill frozen or hung up processes and applications using GUI applications or terminal commands.
+keywords:
+  - frozen applications
+  - frozen desktop
+  - frozen terminal
+  - hung up
+  - not responding
+  - System76
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section:
+---
+
+## Closing Frozen Processes, Applications, and Desktops
+
+Linux offers many methods to stop hanging or frozen processes. The System Monitor provides a GUI process management application with features similar to the Windows Task Manager, while the `kill` command can end single or multiple processes from the Terminal.
```

Comment:
Same thing as before regarding "The System Monitor" (it was corrected in the section below, but it should probably be consistent here too.)